pymchelper/io.py

```diff
@@ -107,15 +103,15 @@ def _get_detector(filelist, error, nan):
         # joblib Parallel class expects the verbosity as a larger number (i.e. multiple of 10)
         worker = Parallel(n_jobs=jobs, verbose=verbose * 10)
         result = worker(
-            delayed(_get_detector)(filelist, error, nan)
+            delayed(fromfilelist)(filelist, error, nan)
             for core_name, filelist in core_names_dict.items()
         )
     except (ImportError, SyntaxError):
         # single-cpu implementation, in case joblib library fails (i.e. Python 3.2)
         logger.info("Single CPU processing")
-        result = [_get_detector(core_name, filelist)
+        result = [fromfilelist(filelist, error, nan)
                   for core_name, filelist in core_names_dict.items()]
-        return result
+    return result
```

Comment:
I'm not sure if this will be always assigned, but at least we'll get error and a clue where to look.